modules/core/src/main/java/org/apache/synapse/mediators/builtin/ValidateMediator.java

```diff
@@ -563,6 +566,27 @@ public boolean mediate(MessageContext synCtx,
         return result;
     }
 
+    /**
+     * Initialize child mediators recursively
+     *
+     * @param se synapse environment
+     */
+    public void init(SynapseEnvironment se) {
+        if (Boolean.parseBoolean(se.getSynapseConfiguration().getProperty(SYNAPSE_VALIDATE_MEDIATOR_REDEPLOYMENT_CACHE_CLEAR))){
```

Comment:
shall we add false as default value

Reply:
It already provides boolean false as the default value